sunpy/net/hek2vso/hek2vso.py

```diff
@@ -150,8 +148,10 @@ def full_query(self, client_query, limit=None, progress=False):
         self.hek_results = self.hek_client.search(*client_query)
         self._quick_clean()
         return self.translate_and_query(self.hek_results,
-                                        limit=limit, progress=progress)
+                                        limit=limit)
 
+    @deprecated_renamed_argument("vso_response_format", None,"6.0", warning_type=SunpyDeprecationWarning)
+    @deprecated_renamed_argument("progress", None,"6.0", warning_type=SunpyDeprecationWarning)
```

Comment:
The other choice is to pass progress down so it works?

Reply:
Why would we deprecate it in `full_query` but not here? Maybe I'm missing some context

Reply:
I mean more, I could work out how where progress should be passed and then not deprecate it across everything.